internal/connector/pingone/platform/resources/pingone_key_test.go

```diff
@@ -0,0 +1,42 @@
+package resources_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/pingone/platform/resources"
+	"github.com/pingidentity/pingctl/internal/testutils"
+)
+
+func TestKeyExport(t *testing.T) {
+	// Get initialized apiClient and resource
+	sdkClientInfo := testutils.GetPingOneSDKClientInfo(t)
+	resource := resources.Key(sdkClientInfo)
+
+	// Defined the expected ImportBlocks for the resource
+	expectedImportBlocks := []connector.ImportBlock{
+		{
+			ResourceType: "pingone_key",
+			ResourceName: "PingOne SSO Certificate for PingFederate Terraform Provider environment_ENCRYPTION",
```

Comment:
Can terraform resource names have spaces? Or is this just because this happens before the import block gets sanitized?

Reply:
This is before sanitation yes.